pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -640,9 +640,10 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 			//FIXME: gracePeriod is nil
 			err = rpDeployment.Stop(namespace, t.Name, TIMEOUT*time.Second, nil)
 			if err != nil {
-				return err
```

Comment:
How about we `defer` the `return err` statements? This was, the errors will get returned in the end of `Undeploy()` and by that time the error resources would also have been deleted?